<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="inputs">
     <attribute name="title">
      <string>Inputs</string>
     </attribute>
     <widget class="QSlider" name="minimum_age">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="pageStep">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="slider_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum Age</string>
      </property>
     </widget>
     <widget class="QSlider" name="maximum_age">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="pageStep">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="slider_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum Age</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="minimum_set_subset">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>48</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="spin_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimum set/subset</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="radio_group">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Analysis Selection</string>
      </property>
      <widget class="QRadioButton" name="analysis_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Analysis 1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="analysis_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Analysis 2</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="line_edit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="edit_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial Last Name Letter</string>
      </property>
     </widget>
     <widget class="QLabel" name="min_value">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="max_value">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="outputs">
     <attribute name="title">
      <string>Outputs</string>
     </attribute>
     <widget class="QPushButton" name="query_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Do query</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="query_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>571</width>
        <height>341</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>100</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
